### ISSUE TITLE
How can I get the latest result of 47.4? and overfitting problem

### ISSUE THREAD

**Issue by Zhou-Renjie** [aharley/bev_baseline#62]
I used the latest code and the training configuration in the README and got a result of 47.2, which is different from the latest result of 47.4. And from the visualization of tensorboard, I can see that there is overfitting in the later stage of training. How can I get the latest result of 47.4?

**Comment by Zhou-Renjie** [comment]
I downloaded the model using get_rgb_model.sh, and the evaluation result is also 47.2. I am not sure if it is a problem with the validation set.

**Comment by aharley** [comment]
My first guess is that this is a GPU-dependent issue. I've seen some variance across gpus & machines that I can't really explain. Do you have another machine you can try this on?
